.github/workflows/integration-test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       matrix:
         # TODO: readd macos-latest once we fix the integration tests
         os: [ ubuntu-latest, windows-latest ]
-        python-version: [ 3.7, 3.8, 3.9 ]
+        python-version: [3.7, 3.8, 3.9, '3.10']
```

Comment:
One more thing: we also need to update `release-cloudbuild.yml` to build images for python 3.10.

Reply:
Good point, I thought of this a few times but never got around to doing it!